Tensile/Source/lib/include/Tensile/hip/HipUtils.hpp

```diff
@@ -69,6 +69,14 @@
         }                                                                                         \
     } while(0)
 
+#define HIP_REPORT_EXC(expr)   \
```

Comment:
`EXC` in this context stands for 'exception'.  A better name for this macro would be `HIP_CHECK_RET` or `HIP_CHECK_RETURN` since it does not throw an exception.

Reply:
@sdquiring Thanks for your suggestion. Done.